package.json

```diff
@@ -97,6 +97,7 @@
     "mocha": "^8.1.3",
     "nyc": "^15.1.0",
     "prompt": "^1.0.0",
+    "regenerator-runtime": "^0.13.7",
```

Comment:
This was needed for the async/await in the tests for the 'returns correct auth url with all combinations of valid input' test. Since that test loops through a bunch of different scenarios, I couldn't find a solid way to test it except with async/await which fails unless regenerator-runtime is imported

Reply:
I would really prefer not to include more dependencies if possible. From what I saw we only have 1 test that uses async await logic and it looks like it could just be converted into the normal promise based logic. Can we do that instead to avoid this dependency?